### PR TITLE
Add Richardson number diagnostic

### LINKED ISSUE #81: Richardson number diagnostic
Would be useful to add Ri(z) to the long list of statistics that are output.

It can be computed from U(z), V(z), and B(z) but probably we need to compute the local Ri(x, y, z) then horizontally average?

@tomchor also wrote a `KernelComputedField` in Oceanostics.jl: https://github.com/tomchor/Oceanostics.jl/blob/81d48c323f38f38617ed162e320d14c1691a615f/src/FlowDiagnostics.jl#L7-L15

Do we just add a dependency on Oceanostics.jl to make use of all the new diagnostics?

cc @xkykai

### PR DESCRIPTION
**Pull request by ali-ramadhan**
Thanks @tomchor for Oceanostics.jl!

Resolves #81

Note: This PR depends on #85.

### CODE REVIEW

**Comment by tomchor** [comment]
Per the discussion in https://github.com/tomchor/Oceanostics.jl/issues/1, it's best to wait until the signature of `richardson_number_ccf!` changes, before merging, no?

**Comment by ali-ramadhan** [comment]
I'm happy to wait to make the change before merging. Otherwise I think the script will break on master without a future PR.

**Comment by tomchor** [comment]
@ali-ramadhan Are you going to make that changes in `Oceanostics.jl` or should I? I might take a little while until I'm able to do it

**Comment by ali-ramadhan** [comment]
I might be able to but maybe not for a couple of days.

I'll revisit this PR then and if the change hasn't been made I can open a PR on Oceanostics.jl.

**Comment by tomchor** [comment]
Alright, leave it with me then! I think I can do it faster.

File: examples/three_layer_constant_fluxes.jl [hunk 1]
```diff
@@ -391,9 +393,16 @@ dissipation = ViscousDissipation(model, data=ccc_scratch.data)
 tke_budget_statistics = turbulent_kinetic_energy_budget(model, b=b, p=p, U=U, V=V, e=e,
                                                         shear_production=shear_production, dissipation=dissipation)
 
+Ri_local = KernelComputedField(Center, Center, Face, richardson_number_ccf!, model,
+                               field_dependencies=(U, V, b), parameters=(dUdz_bg=0, dVdz_bg=0, N2_bg=0))
+
+Ri = AveragedField(Ri_local, dims=(1, 2))
+
+extra_statistics = Dict(:Ri => Ri)
```
Comment (glwagner) [inline examples/three_layer_constant_fluxes.jl:401]: Maybe... "dynamics_statistics"?

**Review by glwagner — APPROVED** [review]

**Pull request merged**